validate_test.go

```diff
@@ -1,14 +1,25 @@
 package agents
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 )
 
 func TestPatterns(t *testing.T) {
-	for i, crawler := range Crawlers {
+	// loading all crawlers wwith go:embed
+	// some validation happens in UnmarshalJSON
+	allCrawlers := Crawlers
+
+	// there is at least 10 crawlers
+	require.GreaterOrEqual(t, len(allCrawlers), 10)
+
+	for i, crawler := range allCrawlers {
 		t.Run(crawler.URL, func(t *testing.T) {
+			// print pattern to console for quickcheck in CI
+			fmt.Print(crawler.Pattern)
```

Comment:
Use fmt.Println to print each pattern on a separate line.

Also maybe it is better to put crawler.Pattern as subtest name (first argument of t`.Run`) and run with `go test -v`, it will print each subtest name (which would be a pattern).

Reply:
great idea, could you do it?

Reply:
Done. I pushed to the branch.